### ISSUE TITLE
Failed to mount component: template or render function not defined

### ISSUE THREAD

**Issue by dawgavins** [xaksis/vue-good-table#21]
I'd like to use this component (looks nice), but I have run into an issue.  When I try to load the page using the component, I get this:

```
[Vue warn]: Failed to mount component: template or render function not defined.

found in

---> <VueGoodTable>
       <Reports> at /Users/User/Documents/project/x3/resources/assets/js/components/Reports.vue
         <MainPage> at /Users/User/Documents/project/x3/resources/assets/js/components/MainPage.vue
```

I am using other vue components installed via NPM successfully.  I'm using a very recent install of Homestead, so Laravel 5.4 with Vue 2.1.10, Webpack, Laravel Mix. 

I've googled it but found nothing that fixes it so far.  So I figured it would be worth posting here.

Thanks!

**Comment by dawgavins** [comment]
Follow-up:  if I just use
`
    import VueGoodTable from 'vue-good-table/src/components/table.vue';
`
the component is working.

**Comment by xaksis** [comment]
hmm this is strange. I haven't been able to reproduce this yet. Will continue digging.

**Comment by dawgavins** [comment]
Ok, let me know if I can provide any more info or help.

**Comment by xaksis** [comment]
@dawgavins can you post some code as to how you're trying to load it into your app when you see that error?

**Comment by cristijora** [comment]
@dawgavins if you `import VueGoodTable from 'vue-good-table` you have to install it as a plugin
`Vue.use(VueGoodTable)` 
Directly using the component without installing it (Vue.use) will not work.

**Comment by xaksis** [comment]
@cristijora with the answers! Thanks! will update.

**Comment by dawgavins** [comment]
Sorry for the delay in responding.  @cristijora nailed it....the local usage posted at the bottom was exactly what I was trying to do.  

I verified that it also works if I use 
`Vue.use(VueGoodTable)`

I had tried that before, but I think I might have left something else in my code (possibly listing VueGoodTable as a local component) that caused an error.  But it does work, and is probably cleaner than specifying the path to the component as per my first workaround.  Of course cristijora's suggestion will be the best overall once its in.

Thanks everyone!
